reactor-core/src/main/kotlin/reactor/core/publisher/MonoFunctions.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2011-2018 Pivotal Software Inc, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:JvmName("MonoWhenFunctionsKt") // TODO Remove in next major version
+package reactor.core.publisher
+
+import org.reactivestreams.Publisher
+
+/**
+ * Aggregates this [Iterable] of void [Publisher]s into a new [Mono].
+ * An alias for a corresponding [Mono.when] to avoid use of `when`, which is a keyword in Kotlin.
+ *
+ * TODO Move to MonoExtensions.kt in next major version
```

Comment:
maybe not something for this PR, but for 3.3.0, we should fix this TODO, sounds like a perfect timing for it.